docs/docs/ref-02-component-api.md

```diff
@@ -66,11 +66,11 @@ Like `setState()` but deletes any pre-existing state keys that are not in nextSt
 forceUpdate([function callback])
 ```
 
-If your `render()` method reads from something other than `this.props` or `this.state`, you'll need to tell React when it needs to re-run `render()` by calling `forceUpdate()`. You'll also need to call `forceUpdate()` if you mutate `this.state` directly.
+By default, updating your component's `props` or `state` will cause your component to re-render. However, if these change implicitly (eg: data deep within an object changes without changing the object itself) or if your `render()` method depends on some other data, you can tell React that it needs to re-run `render()` by calling `forceUpdate()`.
```

Comment:
hmm yeah, I intentionally avoided saying `this.props` because I was afraid of that too. But I was hoping to mention props because I wanted to hammer home those two things making your component pure. How about "By default, when your component's state or props change, your component will re-render."?

Or would you prefer to kill that first sentence and `s/these/state`?